exports/dataset.cpp

```diff
@@ -1238,7 +1308,7 @@ PYBIND11_MODULE(dataset, m) {
       .def("__rmul__",
            [](const Dataset &self, double &other) { return self * other; },
            py::is_operator())
-      .def("merge", &Dataset::merge)
+      .def("merge", &Dataset::merge, "Merge two Datasets together.")
```

Comment:
Can you be more specific about `merge` and `concatenate`. The meanings could be confused?